<?xml version="1.0"?>
<launch> 

  <arg name="path_file" default="/home/irobot/data.yml"/>
  <arg name="id_marker" default="0"/>
  <arg name="size_marker" default="0.20"/>
  <arg name="camera_topic" default="/summit_xl/rear_realsense"/>
  <arg name="camera_name" default="rear"/>



  <!-- Homography calibration -->
  <node pkg="homography_calibration" type="homography_calibration" name="homography_calibration" output="screen">
    <param name="path_file" value="$(arg path_file)"/>
    <param name="id_marker" value="$(arg id_marker)"/>
    <param name="size_marker" value="$(arg size_marker)"/>
    <param name="camera_topic" value="$(arg camera_topic)"/>
    <param name="camera_name" value="$(arg camera_name)"/>

  </node>  

</launch>


